File: frontend/js/src/app/apophthegm.ts
import { OrigText } from "./orig-text";
import { RegText } from "./reg-text";

export class Apophthegm {

    constructor() {
        this.id = "";
        this.textId = "";
        this.orig = new OrigText;
        this.reg = new RegText;
        this.relations = [];
    }
    id : String
    textId : String
    orig : OrigText
    reg : RegText
    relations : String[]

    
}

File: src/main/java/com/example/poems_app/xml/Apophthegm.java
package com.example.poems_app.xml;

import java.util.List;

public class Apophthegm {
	private Orig orig;
	
	private Reg reg;
	
	private String textId;
	
	private List<String> relations;

	public Orig getOrig() {
		return orig;
	}

	public void setOrig(Orig orig) {
		this.orig = orig;
	}

	public Reg getReg() {
		return reg;
	}

	public void setReg(Reg reg) {
		this.reg = reg;
	}
	
	public String getTextId() {
		return textId;
	}

	public void setTextId(String textId) {
		this.textId = textId;
	}
	
	public List<String> getRelations() {
		return this.relations;
	}

	public void setRelations(List<String> relations) {
		this.relations = relations;
	}

}
